build/Makefile.kits

```diff
@@ -93,8 +97,13 @@ else
 	# Build for COMBINED_PACKAGES (.rpm and .deb living together, mass hysteria)
 	# First comes RPM packages
 	sudo rm -rf $(STAGING_DIR)
+ifeq ($(PF_ARCH),x64)
+	rm -rf $(INTERMEDIATE_DIR)/{098,100,110}
+	mkdir -p $(INTERMEDIATE_DIR)/{098,100,110}
+else
 	rm -rf $(INTERMEDIATE_DIR)/{098,100}
 	mkdir -p $(INTERMEDIATE_DIR)/{098,100}
+endif
```

Comment:
I'd leave these unchanged. Then add lines for just `110` down when you already have an `ifeq` at line 144. A nit, but keeps things cleaner.